### ISSUE TITLE
Unable to start

### ISSUE THREAD

**Issue by crosofg** [andersbll/neural_artistic_style#50]
```
python neural_artistic_style.py --subject images/tuebingen.jpg --style images/starry_night.jpg

```

```
Traceback (most recent call last):
  File "neural_artistic_style.py", line 138, in <module>
    run()
  File "neural_artistic_style.py", line 102, in run
    style_img = imread(args.style) - pixel_mean
TypeError: ufunc 'subtract' did not contain a loop with signature matching types dtype('<U32') dtype('<U32') dtype('<U32')
```

**Comment by DylanAlloy** [comment]
Also happening on GTX 1080 _without CUDA_ (I would prefer to have CUDA but seeing as other deep learning installations are working wonderfully and I've followed all (3 lines) of instructions, I will have to suppose that CUDA on this is largely broken and andersbll hasn't accepted PRs or been online in a few months)

However, just like you, I ran into this error using NumPy. Looks like for now the repo is broken.

**Comment by mxchinegod** [comment]
Same error for me. Python 3. Cython 0.19

Before that error, it also doesn't recognize the CUDA installation. I followed the instructions from Deeppy and Cudarray with my env variables but still no luck. 1080GTX, CUDA 8.0RC, 367.44 drivers.

**Comment by chenhd** [comment]
Same error for me. 
imread(args.style).dtype is dtype('float64'),
pixel_mean.dtype is dtype('<U9').

**Comment by lioning2016** [comment]
Same error, i checked and imported deeppy and cudarray, there was no error.So, it should be error in the vgg.mat. I found the vgg19 that has beta10 13 16 and 22 old versions. There are different structures. So i think, that is the point. Ask author  which the real vgg19(MD5 check). 49e623de543b207d57fab0f6eaf79a7e

**Comment by andersbll** [comment]
Hi all, thank you for the feedback and sorry for the lateness of my reply. This is now fixed, see #51.

Best, Anders
